docs/source/configuration/metrics.mdx

```diff
@@ -65,6 +65,7 @@ The following metrics are available using Prometheus:
 - Number of cache misses for different `kind` of cache (`apq`, `query planner`, `introspection`) and for different `storage` (`memory`, `redis`): `apollo_router_cache_miss_count`
 - Time to hit the cache for different `kind` of cache (`apq`, `query planner`, `introspection`) and for different `storage` (`memory`, `redis`): `apollo_router_cache_hit_time`
 - Time to miss the cache for different `kind` of cache (`apq`, `query planner`, `introspection`) and for different `storage` (`memory`, `redis`): `apollo_router_cache_miss_time`
+- Time spent processing a request, outside of waiting for external or subgraph requests (`apollo_router_processing_time`)
```

Comment:
Could you add info to indicate the unit is in seconds please ?

Reply:
done in [7afe859](https://github.com/apollographql/router/pull/2371/commits/7afe859dc926c0a9ca9c4c74a293f0fa16b6516e)